model/service.go

```diff
@@ -54,6 +54,9 @@ type Service struct {
 	// service DNS name.  External services are name-based solution to represent
 	// external service instances as a service inside the cluster.
 	ExternalName string `json:"external"`
+
+	// ServiceAccounts specifies the service accounts that run the service.
+	ServiceAccounts []string `json:"serviceaccounts,omitempty"`
```

Comment:
@kyessenov I'm a little confused - is there a service instance for each VM? How does the operator specify the service account for each VM service instance?